react/modules/tasks/tests.md

```diff
@@ -18,14 +18,79 @@ You must use **Jest** or **Vitest** as the test runner and **React Testing Libra
 - Practice testing user interactions and component state changes
 - Prepare a solid test foundation for future refactoring (**Week 3 - Functional Routing**)
 
-## Application Requirements
+## Functional Requirements (max **100 points**)
 
-1. **Test Coverage**: Achieve at least **80% statement coverage** for all components, with minimum 50% coverage for branches, functions, and lines.
-2. **No Implementation Changes**: Write tests for existing class components without modifying their code
-3. **Behavior-Focused Testing**: Test what components do, not how they do it
-4. **API Mocking**: Mock all external API calls to ensure tests are isolated and fast
-5. **Error Handling**: Test both success and error scenarios
-6. **User Interactions**: Test all user interactions (clicks, form submissions, input changes)
+### Feature 1: Test Coverage (**20 points**)
+
+**As a** developer
+**I want** to ensure high test coverage
+**So that** my components are reliably tested
+
+**Scenario:** Achieve minimum coverage
+
+- **Given** I have written tests for all components
+- **Then** statement coverage is at least 80%
+- **And** branch, function, and line coverage are at least 50%
```

Comment:
Could you, please, add acceptance criterias, too?
E.g. https://github.com/rolling-scopes-school/tasks/blob/master/react/modules/tasks/class-components.md